src/engraving/libmscore/excerpt.cpp

```diff
@@ -114,7 +114,7 @@ bool Excerpt::isEmpty() const
     return partScore() ? partScore()->parts().empty() : true;
 }
 
-void Excerpt::removePart(const QString& id)
+void Excerpt::removePart(const ID& id)
```

Comment:
Would it be better to call it StaffID or PartID to make it clear what sort of ID it is elsewhere? (I'd think the argument name `id` would be better if it was more explicit too)

Reply:
I don't like the idea to create aliases for all sorts of IDs (PartId, StaffId, MeasureId...). I think, the best solution here (and in the same situation) is just to rename the argument to partId):

void Excerpt::removePart(const ID& partId)